public/app/features/org/OrgProfile.tsx

```diff
@@ -15,19 +16,20 @@ interface FormDTO {
 
 const OrgProfile = ({ onSubmit, orgName }: Props) => {
   const canWriteOrg = contextSrv.hasPermission(AccessControlAction.OrgsWrite);
+  const initialFormModel: FormDTO = { orgName };
+
+  const { handleSubmit, register } = useForm<FormDTO>({ defaultValues: initialFormModel });
 
   return (
-    <Form defaultValues={{ orgName }} onSubmit={({ orgName }: FormDTO) => onSubmit(orgName)}>
-      {({ register }) => (
-        <FieldSet label="Organization profile" disabled={!canWriteOrg}>
-          <Field label="Organization name">
-            <Input id="org-name-input" type="text" {...register('orgName', { required: true })} />
-          </Field>
-
-          <Button type="submit">Update organization name</Button>
-        </FieldSet>
-      )}
-    </Form>
+    <form name="organizationProfile" onSubmit={handleSubmit((form) => onSubmit(form.orgName))}>
```

Comment:
```suggestion
    <form name="organizationProfile" onSubmit={handleSubmit((form) => onSubmit(form.orgName))}
       style={{ width: '100%', maxWidth: '600px' }}
    >
```
This will need some styles to be compatible with the removed component.